python/ray/tune/utils/placement_groups.py

```diff
@@ -0,0 +1,227 @@
+from typing import Dict, Optional, Set
+import os
+import time
+
+import ray
+from ray import ObjectRef
+from ray.actor import ActorClass
+from ray.tune.trial import Trial
+from ray.util.placement_group import PlacementGroup
```

Comment:
@rkooo567 take a look! this should help inform future placement group APIs

cc @ericl

Reply:
This is a really great idea! I also thought the placement group should be eventually hided behind the highe-level APIs. (maybe something like Group APIs, e.g., GangSchedulingGroup, LoadBalancingGroup).